File: example/ApiTest/Python/documents/1_upload_documents.py
# request-----------------------------------------------
import requests

url = "http://127.0.0.1/api/v1/datasets/842986a8ad5711ef8b530242ac120003/documents"

payload = {}
files=[
  ('file',('hd.txt',open('D:/资料/ragflow/hd.txt','rb'),'text/plain')),
  ('file',('数字中国建设整体布局规划.txt',open('D:/资料/ragflow/数字中国建设整体布局规划.txt','rb'),'text/plain'))
]
headers = {
  'Authorization': 'Bearer ragflow-hjNzA4ODI4YWM5MTExZWY5YzUyMDI0Mm'
}

response = requests.request("POST", url, headers=headers, data=payload, files=files)

print(response.text)

File: example/ApiTest/JavaScript/documents/1_upload_documents.js
const myHeaders = new Headers();
myHeaders.append("Authorization", "Bearer ragflow-hjNzA4ODI4YWM5MTExZWY5YzUyMDI0Mm");

const formdata = new FormData();
formdata.append("file", fileInput.files[0], "D:/ragflow/hd.txt");
formdata.append("file", fileInput.files[0], "D:/ragflow/测试.txt");

const requestOptions = {
  method: "POST",
  headers: myHeaders,
  body: formdata,
  redirect: "follow"
};

fetch("http://127.0.0.1/api/v1/datasets/8a85ab34ad5311ef98b00242ac120003/documents", requestOptions)
  .then((response) => response.text())
  .then((result) => console.log(result))
  .catch((error) => console.error(error));